scripts/create-package.sh

```diff
@@ -0,0 +1,243 @@
+            readme_tmpl="${pkg_dir}/README.txt"
+            ;;
+    esac
+    # Fill template variables in README.template
+    [ -n "$git_commit" ] && sed -i -e "s|%GIT_COMMIT%|$git_commit|" "$readme_tmpl"
+    [ -n "$git_branch" ] && sed -i -e "s|%GIT_BRANCH%|$git_branch|" "$readme_tmpl"
+    [ -n "$git_repo" ]   && sed -i -e "s|%GITHUB_REPO%|$git_repo|"  "$readme_tmpl"
+}
+
+install_translation()
+{
+    lng_dir=${pkg_dir}/translations
+    if [ "$platform" = "macos" ]; then
+        lng_dir=${macos_dst_dir}/Resources/translations
+    fi
+    # Prepare translation files
+    #
+    # Note:
+    #   We conciously drop the dialect postfix because no dialects are available.
+    #   (US was the default DOS dialect and therefore is the default for 'en').
+    #   There users get the generic translation and benefit from simpler filenames.
+    #   Dialect translations will be added if/when they're available.
+    #
+    install_file contrib/translations/de/de_DE.lng       "$lng_dir/de.lng"
+    install_file contrib/translations/en/en_US.lng       "$lng_dir/en.lng"
+    install_file contrib/translations/es/es_ES.lng       "$lng_dir/es.lng"
+    install_file contrib/translations/fr/fr_FR.lng       "$lng_dir/fr.lng"
+    install_file contrib/translations/it/it_IT.lng       "$lng_dir/it.lng"
+    install_file contrib/translations/pl/pl_PL.CP437.lng "$lng_dir/pl.cp437.lng"
+    install_file contrib/translations/pl/pl_PL.lng       "$lng_dir/pl.lng"
+    install_file contrib/translations/ru/ru_RU.lng       "$lng_dir/ru.lng"
+}
+
+pkg_linux()
+{
+    # Print shared object dependencies
+    ldd "${build_dir}/dosbox"
+    install -DT "${build_dir}/dosbox" "${pkg_dir}/dosbox"
+
+    install -DT contrib/linux/dosbox-staging.desktop "${pkg_dir}/desktop/dosbox-staging.desktop"
+    DESTDIR="$(realpath "$pkg_dir")" make -C contrib/icons/ install datadir=
+}
+
+pkg_macos()
+{
+    # Note, this script assumes macos builds have x86_64 and ARM64 subdirectories
+
+    # Print shared object dependencies
+    otool -L "${build_dir}/dosbox-arm64/dosbox"
+    python3 scripts/verify-macos-dylibs.py "${build_dir}/dosbox-arm64/dosbox"
+
+    # Create universal binary from both architectures
+    mkdir dosbox-universal
+    lipo dosbox-x86_64/dosbox dosbox-arm64/dosbox -create -output dosbox-universal/dosbox
+
+    # Generate icon
+    make -C contrib/icons/ dosbox-staging.icns
+
+    install -d   "${macos_dst_dir}/MacOS/"
+    install      dosbox-universal/dosbox           "${macos_dst_dir}/MacOS/"
+    install_file contrib/macos/Info.plist.template "${macos_dst_dir}/Info.plist"
+    install_file contrib/macos/PkgInfo             "${macos_dst_dir}/PkgInfo"
+    install_file contrib/icons/dosbox-staging.icns "${macos_dst_dir}/Resources/"
+
+    sed -i -e "s|%VERSION%|${dbox_version}|"       "${macos_dst_dir}/Info.plist"
+}
+
+pkg_msys2()
+{
+    install -DT "${build_dir}/dosbox.exe" "${pkg_dir}/dosbox.exe"
```

Comment:
`set -e` will also protect users if they don't have some of these tools (`ldd / ntldd / otool / lipo / .. ), and avoid the need for a dependency-checker function.